roles/kubernetes/README.rst

```diff
@@ -0,0 +1,40 @@
+Kubernetes
+==========
+
+.. versionadded:: 0.1
+
+**NOTES:** This is NOT kubernetes-mesos/Mantl framework, this is a standalone kubernetes cluster.
+
+**WARNING**: Do NOT install on top of existing Mantl cluster as it might brake it!
+
+**REQUIREMENTS:**
+
+- Terraform binaries installed 
+- Ansible 1.9.x +  ( just run ``pip install -r requirements.txt`` ) 
```

Comment:
Maybe we can shorten this to just advising users to run ``pip install -r requirements.txt``, rather than specifying an Ansible version?